changelog.d/20240313_014112_umangapatel123_refactor_into_usereducer.md

```diff
@@ -0,0 +1,4 @@
+### Fixed
+
+- set `file` and `fileName` to null after import is done and change `useState` to `useReducer`
```

Comment:
We should describe here what it means for a CVAT user, not for developer.

Reply:
can you please review now?